src/lib_ccx/params_dump.c

```diff
@@ -18,7 +18,9 @@ void params_dump(struct lib_ccx_ctx *ctx)
 			mprint ("stdin");
 			break;
 		case CCX_DS_NETWORK:
-			if (ccx_options.udpaddr == NULL)
+            if (ccx_options.udpsrc != NULL)
+                mprint ("Network, %s@%s:%d", ccx_options.udpsrc, ccx_options.udpaddr, ccx_options.udpport);
```

Comment:
udpaddr could be null, first check for null or in if add condition

Reply:
I'm not sure I follow. Is `params_dump.c` used without `params.c` being called? Unless I messed up [the parameter parsing](https://github.com/squarebracket/ccextractor/blob/1dd4201496e6eb1e829de4ca164fb0c4c272295d/src/lib_ccx/params.c#L2225), if you have a `udpsrc` you _must_ have a `udpaddr` -- even if it's just `\0` -- so you won't end up in a condition where `udpaddr` is null if `udpsrc` is not also null.

Of course, if there's a path to `params_dump.c` which doesn't first use `params.c`, then yeah it might be null, but I'm assuming that's not the case.